Fix graph routes

deepakunni3 opened the pull request:
Fix implementation of `graph/node/<id>` and `graph/edges/from/<id>` routes.

This PR will change the behavior of `graph/edges/from/<id>`.

Previously this route would,
- fetch edges with relation `subClassOf`, direction `OUTGOING`, and depth of `20`
- fetch edges with relation `subClassOf`, direction `INCOMING`, and depth of `1`
- fetch edges with relation `equivalentClass`, and depth of `1`

This PR simplifies the behavior of `graph/edges/from/<id>` where it now acts as a simple pass through to SciGraph `graph/neighbors` endpoint and makes no assumption about relation, direction or depth.

deepakunni3 committed:
Fix graph routes

Fix implementation of graph/node/<id> and graph/edges/from/<id> routes.

graph/edges/from/<id> now acts as a simple pass through to
SciGraph graph/neighbors endpoint.

kshefchek commented on biolink/api/graph/endpoints/node.py:
```diff
@@ -1,44 +1,74 @@
-        args = parser.parse_args()
-        
-        return sg.graph(id)
+        g = sg.bioobject(id)
```
I favor descriptive variable names (excepting i,j,k,v), any scoping issues using graph?

deepakunni3 replied:
No issues with adding descriptive variable names. I'll use `graph` instead of `g`.

kshefchek commented on biolink/api/graph/endpoints/node.py:
```diff
@@ -1,44 +1,74 @@
+        args = self.parser.parse_args()
+        response = sg.get_response("dynamic/cliqueLeader", q=id, format="json", depth=1)
+        nodes = response.json()['nodes']
+        if len(nodes) == 0:
```
consider

    if not nodes:
        ...

but either is fine

deepakunni3 replied:
Good call 👍

kshefchek commented:
couple of linting considerations but otherwise looks good, +1

deepakunni3 commented:
Thanks @kshefchek 

I'll incorporate the suggested changes.

deepakunni3 committed:
Linting

Use proper variable names where applicable
Use a more pythonic way of checking if a list is empty
